src/components/contact-us/questions/_account-creation-problem-questions.njk

```diff
@@ -0,0 +1,36 @@
+<h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">
+  {{ 'pages.contactUsQuestions.accountCreationProblem.header' | translate }}
+</h1>
+
+<form action="/contact-us-questions" method="post" novalidate>
+
+<input type="hidden" name="_csrf" value="{{csrfToken}}"/>
+<input type="hidden" name="theme" value="{{theme}}"/>
+<input type="hidden" name="subtheme" value="{{subtheme}}"/>
+<input type="hidden" name="backurl" value="{{backurl}}"/>
```

Comment:
if this is the referrer does it need to be posted back?

Reply:
Added the `backurl` as a hidden field here so when the page fails validation it still knows where the previous page was and can go back there.